components/support/Category.tsx

```diff
@@ -14,28 +14,31 @@ interface Props {
   clickHandler?: (event: React.MouseEvent<HTMLButtonElement>) => void
 }
 
-const category_list = [
-  { type: 'career', image: career, text: 'Career' },
-  { type: 'healthcare', image: healthcare, text: 'Healthcare' },
-  { type: 'financial', image: financial, text: 'Financial Assistance' },
-  { type: 'self', image: self, text: 'Self Development' },
-  { type: 'family', image: family, text: 'Family Services' },
-  { type: 'children', image: children, text: 'Children Education' },
-  { type: 'legal', image: legal, text: 'Legal' }
-]
+const category_dict = {
+  career: [career, 'Career'],
+  healthcare: [healthcare, 'Healthcare'],
+  financial: [financial, 'Financial Assistance'],
+  self: [self, 'Self Development'],
+  family: [family, 'Family Services'],
+  children: [children, 'Children Education'],
+  legal: [legal, 'Legal']
+}
 
 export default function Category(props: Props) {
   const type = props.type
-  const category_ref = category_list.filter(
-    (category) => category.type === type
-  )[0]
+  const category_ref =
+    category_dict[`${type}`] == null
+      ? console.log(
+          'error in Category.tsx: category_ref is undefined as category type does not exist.'
+        )
+      : category_dict[`${type}`]
 
   return (
```

Comment:
I think the simplest way to handle this error would be something like this (sorry if my typescript syntax is incorrect):

```
if (category_dict[`${type}`] == null) {
    console.log('error in Category.tsx: category_ref is undefined as category type does not exist.')
} else {
    return (
       <div>
       ...
    }
}
```

Reply:
ahh i understand now, will push the changes soon. i'm not familiar with writing tests but will look into that too. thanks!